squirrel.worker/src/main/java/org/dice_research/squirrel/analyzer/impl/RDFAnalyzer.java

```diff
@@ -86,25 +86,24 @@ public RDFAnalyzer(UriCollector collector) {
 
 //    @Override
     public boolean isElegible(CrawleableUri curi, File data) {
-        Tika tika = new Tika();
         // Check the content type first
         String contentType = (String) curi.getData(Constants.URI_HTTP_MIME_TYPE_KEY);
         if ((contentType != null) && (contentType.equals("application/rdf+xml") || contentType.equals("text/plain")
-                || contentType.equals("application/x-turtle"))) {
+                || contentType.equals("application/x-turtle") || contentType.equals("application/n-triples"))) {
```

Comment:
We should get this list from the Jena library which we are using for parsing. A piece of code which gathers the available content types can be found at https://github.com/dice-group/ldcbench/blob/develop/ldcbench.http-node/src/test/java/org/dice_research/ldcbench/nodes/http/simple/ContentNegotiationTest.java#L44-L51

Reply:
Please note that it is sufficient to gather them once which the Analyzer is constructed. Later on, it would be sufficient to check whether the given content type is within the generated set.